devserver.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
```

Comment:
Maybe we could use internal `http` module instead of `express` to get rid of extra deps?

We have a similar [dev server](https://github.com/editor-js/image/blob/master/dev/server.js) at the Image tool

Reply:
ok, i'll take a look